### PR TITLE
Document the availability of graph.byelementid on standard databases

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1120.surge.sh

Updated pages:
- [Additional Cypher features](https://neo4j-docs-cypher-1120.surge.sh/cypher-manual/5/appendix/gql-conformance/additional-cypher)
- [USE](https://neo4j-docs-cypher-1120.surge.sh/cypher-manual/5/clauses/use)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1120.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Graph functions](https://neo4j-docs-cypher-1120.surge.sh/cypher-manual/5/functions/graph)

File: modules/ROOT/pages/clauses/use.adoc [hunk 1]
```diff
@@ -114,12 +114,18 @@ USE graph.byName($graphName)
...
 === Query a composite database constituent using elementId
 
-The `graph.byElementId()` function (introduced in Neo4j 5.13), can be used in the `USE` clause to resolve a constituent graph to which a given element id belongs.
-In the below example, it is assumed that the DBMS contains a composite database constituent, which contains the element id `4:c0a65d96-4993-4b0c-b036-e7ebd9174905:0`.
-If the constituent database is not a standard database in the DBMS an error will be thrown:
+The xref:functions/graph.adoc#functions-graph-by-elementid[`graph.byElementId()`] function can be used in the `USE` clause to resolve a constituent graph to which a given xref:functions/scalar.adoc#functions-elementid[element id] belongs.
+As of Neo4j 5.26, it is supported on both standard and composite databases (on previous versions it is only available on composite databases).
+
+[NOTE]
+A USE clause with `graph.byElementId()` cannot be combined with other `USE` clauses unless the subsequent `USE` clauses reference the same element id.
```
Comment (LinneaAndersson) [inline modules/ROOT/pages/clauses/use.adoc]:
```suggestion
On a standard database, a USE clause with `graph.byElementId()` cannot be combined with other `USE` clauses unless the subsequent `USE` clauses reference the same element id.
```

File: modules/ROOT/pages/functions/graph.adoc [hunk 2]
```diff
@@ -192,6 +198,10 @@ For more information, see xref:subqueries/call-subquery.adoc#import-variables[CA
 .Considerations
 |===
 | If the constituent database is not a standard database in the DBMS, an error will be thrown.
+| `graph.byElementId()` is only supported in the xref:clauses/use.adoc[`USE`] clause.
+| As of Neo4j 5.26, `graph.byElementId()` is supported on both link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/[standard and composite databases].
+On earlier versions, it is only supported on composite databases.
+| A `USE` clause with `graph.byElementId()` cannot be combined with other `USE` clauses unless the subsequent `USE` clauses reference the same element id.
```
Comment (LinneaAndersson) [inline modules/ROOT/pages/functions/graph.adoc]:
```suggestion
| On a standard database, a `USE` clause with `graph.byElementId()` cannot be combined with other `USE` clauses unless the subsequent `USE` clauses reference the same element id.
```

**Review by LinneaAndersson — COMMENTED** [review]
Very nice Jens 👍 Just one tiny comment

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by LinneaAndersson — APPROVED** [review]
Thank you very much Jens! 🎉

**Pull request merged**